modules/installation-guide/partials/proc_configuring-che-hostname.adoc

```diff
@@ -29,15 +29,23 @@ $ {orch-cli} create {orch-namespace} {prod-namespace}
 +
 [subs="+quotes,attributes"]
```

Comment:
@max-cx, we should not use 'shell' highlighting on command-line stuff. When you type on the command line, there's no highlighting either. The syntax-highlight feature should only be used when you're listing code. So, for 'shell', it would be for listing the content of shell scripts.

Not sure what you mean by `because \ after $ doesn't render `...

Reply:
I added an explanatory comment to the JIRA issue. TL;DR: the not-rendered backslash is intentional.